packages/webview_flutter/example/test_driver/webview_flutter_e2e.dart

```diff
@@ -536,6 +536,93 @@ void main() {
     expect(title, 'Some title');
   });
 
+  group('Programmatic Scroll', () {
+    testWidgets('setAndGetScrollPosition', (WidgetTester tester) async {
+      final String scrollTestPage = '''
+        <!DOCTYPE html>
+        <html>    
+          <head>
+            <style>
+              html {
+                height: 100%;
+                width: 100%;
+                overflow-x: auto;
+                overflow-y: auto;
+              }
+      
+              body {
+                height: 100%;
+                width: 100%;
+              }
+            </style>
+      
+            <script type="text/javascript">
+              function config() {
+                //make sure scrolling is enabled on both x & y.
```

Comment:
nit: I'm a little confused by the comment, is this make sure scrolling is enabled, or is it making sure that the page's content is big enough to require scrolling?

Reply:
Thanks for note, you're correct. Changed to: "Create a page with dimensions big enough to allow scrolling on both x & y." in coming commit.